patches/chrome-browser-safe_browsing-download_protection-check_client_download_request.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request.cc b/chrome/browser/safe_browsing/download_protection/check_client_download_request.cc
+index 59a28c8c72340677aaed45e042fe8ad499fa4fed..e3fcd558490b53aba2f29421a3d1b1be6f9f2dab 100644
+--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request.cc
++++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request.cc
+@@ -599,6 +599,7 @@ std::string CheckClientDownloadRequest::SanitizeUrl(const GURL& url) const {
+ 
+ void CheckClientDownloadRequest::SendRequest() {
+   DCHECK_CURRENTLY_ON(BrowserThread::UI);
++  BRAVE_DISABLE_DOWNLOAD_PROTECTION_REMOTE_LOOKUPS
```

Comment:
we can block this in the network delegate so no patch is required

Reply:
Is there an example I can copy from?

Reply:
I think you can follow existing patterns like this - https://github.com/brave/brave-core/blob/master/browser/net/brave_common_static_redirect_network_delegate_helper.cc#L48
and then cancelling it.

Reply:
I found the way that we have to [cancel normal network requests](https://github.com/brave/brave-core/blob/a8e3ffbc1ce0b423841925822591fd3fd3c87fea/browser/net/brave_translate_redirect_network_delegate_helper.cc#L64), but for system requests though, it seems we only have a [way to change the URL](https://github.com/brave/brave-core/blob/66639ff699a7449fd3f6b32404ab457a2451b625/patches/chrome-browser-net-system_network_context_manager.cc.patch). @iefremov suggested that I instead replace the URLs with `https://no-thanks.invalid` since that's [used in a bunch of places already](https://github.com/brave/brave-core/search?q=no-thanks.invalid&unscoped_q=no-thanks.invalid).